transform/go/go_gaf2schema.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+# parses GO annotation file
+# http://www.geneontology.org/doc/GO.references
+
+# curl -L -O http://geneontology.org/gene-associations/goa_human.gaf.gz
+# curl -O ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/idmapping/by_organism/HUMAN_9606_idmapping.dat.gz
+
+import sys
+import json
+import gzip
+
+from bmeg.vertex import Gene, GeneOntologyTerm
+from bmeg.edge import GeneOntologyAnnotation
+from bmeg.emitter import JSONEmitter
+
+def uniprot_ns(n):
+	return "uniprot:" + n
+
+def ensembl_ns(n):
+	return "ensembl:" + n
+
+def go_ns(n):
+	return "go:" + n
+
+def goref_ns(n):
+	return "goref:" + n
+
+def gene_ns(n):
+	return "gene:" + n
+
+def pubmed_ns(n):
+	return "pubmed:" + n
+
+UNIPROT_COL = 1
+SYMBOL_COL = 2
+GOID_COL = 4
+REF_COL = 5
+EVIDENCE_COL = 6
+NAME_COL = 9
+
+def message_to_json(handle, message):
+	msg = json_format.MessageToDict(message, preserving_proto_field_name=True)
+	handle.write(json.dumps(msg))
+	handle.write("\n")
+
+if __name__ == "__main__":
+
+	uniprot_2_ensembl = {}
+
+	with gzip.GzipFile(sys.argv[2]) as handle:
```

Comment:
We have some helpers for args and readers:

https://github.com/bmeg/bmeg-etl/blob/models/src/bmeg/ioutils.py#L6

https://github.com/bmeg/bmeg-etl/blob/models/src/bmeg/util/cli.py#L8